ida/data.yml

```
@@ -396,10 +404,25 @@ functions:
   0x140A396B0: ProcessPacketUpdatePartyPositions
   0x140A396E0: ProcessPacketUpdateAllianceNormalPositions
   0x140A39710: ProcessPacketUpdateAllianceSmallPositions
+  0x140A39830: ProcessPacketChatIncoming
+  0x140A399E0: ProcessPacketInventoryTransaction
+  0x140A39A00: ProcessPacketItemInfo
   0x140A39B90: ProcessPacketRSVData
   0x140A39BB0: ProcessPacketRSFData
   0x140A39C10: ProcessPacketHuntingLogEntry
   0x140A39C50: ProcessPacketNPCYell
+  0x140A39D10: ProcessPacketEquipDisplayFlags
+  0x140A39DE0: ProcessPacketCompletedAchievements
+  0x140A39E30: ProcessPacketLandSetInitialize
```

This address resolves to something Achievement related.
It seems to be matching an address in 7.2hf1. Here the correct address for 7.21:
```suggestion
  0x140A3A510: ProcessPacketLandSetInitialize
```